marlowe-apps/src/Language/Marlowe/Runtime/App/Types.hs

```diff
@@ -143,32 +128,7 @@ instance Default Config where
       }
 
 -- | A monad type for Marlowe Runtime.Client programs.
-newtype Client a = Client {runClient :: MarloweT IO a}
-  deriving newtype
-    (Alternative, Applicative, Functor, Monad, MonadBase IO, MonadBaseControl IO, MonadFail, MonadFix, MonadIO)
-
-instance MonadWith Client where
```

Comment:
Do we not need these instances because we're using a type alias now? or is there a deeper reason?

Reply:
It's because I switched it to a type alias - but this was also failing to compile for some reason - hence the switch to the type alias.